foreman/client.py

```diff
@@ -678,7 +680,7 @@ def _get_local_defs(self, strict=True):
         """
         version = parse_version(self.version)
         for cache_dir in [
-            os.path.join(os.path.expanduser('~'), '.python-foreman'),
+            self.cache_dir,
```

Comment:
I'd expect here to still fall back to the user-home cache dir if the given version is not in the specified cache dir.